org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/XMLWorkspaceService.java

```diff
@@ -37,6 +46,25 @@ public XMLWorkspaceService(XMLLanguageServer xmlLanguageServer) {
 	public CompletableFuture<List<? extends SymbolInformation>> symbol(WorkspaceSymbolParams params) {
 		return null;
 	}
+	
+	
+	@Override
+	public CompletableFuture<Object> executeCommand(ExecuteCommandParams params) {

```

Comment:
Add javadoc

Reply:
Yes, good point, will do

Reply:
The interface declaring this method should now have the proper javadoc